src/index.css

```diff
@@ -0,0 +1,28 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+.carousel.carousel-slider {
+	overflow: inherit !important;
+}
+.carousel .control-dots {
+	margin: -30px 0 !important;
+}
+li.dot {
+	background-color: #737373 !important;
+	box-shadow: inherit !important;
+}
```

Comment:
You could use [Tailwind's @apply directive](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply) to get the styles from Tailwind isntead.

Reply:
It will take a while to rewrite everything with directive, but will keep in mind for the future screens